src/ops/convolution.cc

```diff
@@ -0,0 +1,230 @@
+/*******************************************************************************
+* Copyright 2018 Intel Corporation
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+*     http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*******************************************************************************/
+#include <vector>
+
+#include "../../../../src/operator/nn/convolution-inl.h"
+#include "convolution.h"
+
+#include <ngraph/builder/quantization.hpp>
+#include "../ngraph_emitter.h"
+#include "../ngraph_emitter_utils.h"
+#include "../ngraph_nnvm_ops.h"
+#include "../ngraph_sgcompiler_utils.h"
+#include "../ngraph_utils.h"
+
+#if MXNET_USE_MKLDNN == 1
+#include "../../../../src/operator/subgraph/mkldnn/mkldnn_conv-inl.h"
+#endif
+
+namespace ngraph_bridge {
+
+struct ConvInputs {
+  NgraphNodePtr data;
+  NgraphNodePtr filter;
+  NgraphNodePtr bias;
+  ngraph::CoordinateDiff pad;
+  ngraph::Strides stride;
+  ngraph::Strides dilate;
+  size_t groups;
+};
```

Comment:
NBD, but would be be more natural to have the following two functions be class constructors instead?